models/classes.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+
+
+class Product(BaseModel):
+    id: int
+    naziv: str
+    cena: int
+    kolicina: int
+
+
+class Buyer(BaseModel):
+    kupac_id: int
+    ime_kupca: str
+    Prezime_kupca: str
```

Comment:
everything has to be lower case, so prezime_kupca

Reply:
I changed